front/src/modules/ui/dropdown/components/StyledDropdownMenu.tsx

```diff
@@ -16,7 +16,7 @@ export const StyledDropdownMenu = styled.div<{
 
   flex-direction: column;
 
-  overflow: hidden;
+  min-width: ${({ width }) => (width && width > 160 ? width : 160)}px;
```

Comment:
Could this possibly work too? If it does work, might be easier to read:

```css
min-width: 160px;
width: ${({ width }) => width}px;
```

Reply:
@thaisguigon totally right !